aristanetworks/ansible-avd
# Feat(eos_designs): Add platform specific management security entropy source settings.

## Change Summary

Add platform specific management security entropy source settings.

## Related Issue(s)

Fixes #1226 

## Depends on 

PR #3841 

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes
Add `security_entropy_source` to `platform_settings`
```
platform_settings:
     security_entropy_source: <str>
```

## How to test
Input `security_entropy_source` in `platform_settings`, run the playbook/molecule and check the structured_configs for management_security
Input - 

```
platform_settings:
  - platforms: [ default ]
    security_entropy_source: hardware
```
 Config -
```
management security
   entropy source hardware
```

## Checklist

### Repository Checklist
